docs/cli.md

```diff
@@ -822,6 +822,12 @@ null, boolean, byte, ubyte, short, ushort, integer, uinteger, long, ulong, float
 
 In addition decimal(precision,scale) and varchar(length).
 
+#### Show a column's audit information
+
+```bash
+gcli column details --name catalog_postgres.hr.departments.name --audit
```

Comment:
Shall we consider promoting 'audit' to a top-level command?

Reply:
It could be done that way. However, having fewer commands and using flags where possible is better from a good practice point of view. (see https://clig.dev/#arguments-and-flags)  If you rearrange the command this way, it probably makes a bit more sense:
`cli column details --audit --name catalog_postgres.hr.departments.name`

In that, there are different details about an entity that can be displayed, audit information being just one, see the table command for an example with multiple display options.

Reply:
Having fewer commands, each with a set of flags, is also easier for a user to remember.

Reply:
I didn't mean this is a blocker. I was simply thinking that if we want to consolidate
all audit kind of operations to one command, under which a user can specify the objects
to check via flags. My understanding is that auditing, in general, is a cross-cutting
concern, rather than something I want to retrieve from `column details`.

Reply:
The current Java API is written in a way that requires you to know the entry type to get the audit information, and there's not a single generic interface. It also returns reasonably limited basic info with no history.

Reply:
> I didn't mean this is a blocker. I was simply thinking that if we want to consolidate all audit kind of operations to one command, under which a user can specify the objects to check via flags. My understanding is that auditing, in general, is a cross-cutting concern, rather than something I want to retrieve from `column details`.

Good point; I like this idea; It would be better to put all audit related actions together, so that the guy who checks the auditing logs can quickly check all the logs across different entities.

Reply:
Good idea, but unrelated to this PR, I'd raise another issue for it as the Java API would need to change.